packages/ipfs/package.json

```diff
@@ -98,7 +98,7 @@
     "ipfs-http-client": "^42.0.0",
     "ipfs-http-response": "^0.5.0",
     "ipfs-mfs": "^1.0.0",
-    "ipfs-multipart": "^0.3.0",
+    "ipfs-multipart": "file:/Users/paulcowgill/Code/contract-work/3box/js-ipfs/packages/ipfs/.connect-deps-cache/ipfs-multipart-0.3.0-1583254040397.tgz",
```

Comment:
TODO: Switch from local `connect-deps` version of `ipfs-multipart` to the one in this repo now that it's also part of the monorepo.

Reply:
Done!